entrypoint.sh

```diff
@@ -11,7 +11,16 @@ process_input(){
         my_dir="./test_data"
     fi
 
-    if [ -n "$input_paths" ]; then
+    severity_mode="$(echo $severity_mode | tr '[:upper:]' '[:lower:]')"
+
+    if [[ "$severity_mode" != "style" && "$severity_mode" != "info" && "$severity_mode" != "warning" && "$severity_mode" != "error" ]]; then
+        if [ -n "$severity_mode" ]; then
+            echo "Error setting unknown severity mode. Defaulting severity mode to style."
+        fi
+        severity_mode="style"
+    fi
+
+    if [ "$input_paths" != "." ]; then
```

Comment:
This doesn't take into account the case where no path argument is passed. Now if you try ./entrypoint.sh with no argument, it doesn't lint anything. I think we better go back to
 if [ -n "$input_paths" ]